poetry.lock

```diff
@@ -1,88 +1,88 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
```

Comment:
I don't know what the policy is with package upgrades on this repo. But, I think it probably better to only upgrade the package that you changed in the pyproject.toml file. All the tests are passing so it's probably ok, but it's not really related to the task.

Reply:
I re-generated the lockfile after updating the version of black: the dependency and the workflows were mismatched, so I updated the version and re-locked the dependencies so it should be only that. I can revert that change if preferable.